blueprints/install-activate-setup-theme-from-gh-repo/blueprint.json

```
@@ -0,0 +1,61 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"meta": {
+		"title": "Loading, activating, and configuring a theme from a GitHub repository.",
+		"description": "This is a good example of typical steps used on a theme's loading, activation and configuration",
+		"author": "richtabor",
+		"categories": ["theme"]
+	},
+	"landingPage": "/wp-admin/site-editor.php",
+	"login": true,
+	"features": {
+		"networking": true
+	},
+	"steps": [
+		{
+			"step": "resetData"
+		},
+		{
+			"step": "writeFile",
+			"path": "/wordpress/wp-content/mu-plugins/rewrite.php",
+			"data": "<?php /* Use pretty permalinks */ add_action( 'after_setup_theme', function() { global $wp_rewrite; $wp_rewrite->set_permalink_structure('/%postname%/'); $wp_rewrite->flush_rules(); } );"
+		},
+		{
+			"step": "updateUserMeta",
+			"meta": {
+				"first_name": "Rich",
+				"last_name": "Tabor",
+				"admin_color": "modern"
+			},
+			"userId": 1
+		},
+		{
+			"step": "installTheme",
+			"themeZipFile": {
+				"resource": "url",
+				"url": "https://github-proxy.com/proxy/?repo=richtabor/kanso&branch=main"
```

Could we load this from the WordPress theme directory instead of GitHub? 

```suggestion
				"resource": "wordpress.org/themes",
				"slug": "kanso"
```

---

Oh, my request makes no sense since your entire point was to showcase using GitHub. Cool! Merging. Thank you! Cross-referencing https://github.com/WordPress/wordpress-playground/issues/1467 and https://github.com/WordPress/wordpress-playground/issues/1708 so we can come back here and replace the github-proxy.com reference with direct checkout when we support that.